Spoj1Divisor Summation/Program.cs

```diff
@@ -7,26 +7,85 @@
 
 namespace Spoj1Divisor_Summation
 {
-    public class Program
-    {
-        public static void Main()
-        {
-            var reader = new StreamReader(Console.OpenStandardInput());
-            var writer = new StreamWriter(Console.OpenStandardOutput());
-            writer.AutoFlush = false;
-
-            DivisorSumation(reader, writer);
+    public class Program
+    {
+        public static void Main()
+        {
+            try
+            {
+                Console.WriteLine("Input:");
+                var reader = new StringReader(NumberInput());

```

Comment:
Chciałbym żeby  reader przyjmował "Console.OpenStandardInput()", a jeśli potrzebujesz operować na konsoli, to operuj na zmiennej 'reader'